homeassistant/components/lg_thinq/sensor.py

```diff
@@ -255,9 +261,93 @@
         translation_key=ThinQProperty.WATER_TYPE,
     ),
 }
+ELAPSED_DAY_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.ELAPSED_DAY_STATE: SensorEntityDescription(
+        key=ThinQProperty.ELAPSED_DAY_STATE,
+        native_unit_of_measurement=UnitOfTime.DAYS,
+        translation_key=ThinQProperty.ELAPSED_DAY_STATE,
+    ),
+    ThinQProperty.ELAPSED_DAY_TOTAL: SensorEntityDescription(
+        key=ThinQProperty.ELAPSED_DAY_TOTAL,
+        native_unit_of_measurement=UnitOfTime.DAYS,
+        translation_key=ThinQProperty.ELAPSED_DAY_TOTAL,
+    ),
+}
+TIME_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    TimerProperty.LIGHT_START: SensorEntityDescription(
+        key=TimerProperty.LIGHT_START,
+        translation_key=TimerProperty.LIGHT_START,
+    ),
+    TimerProperty.ABSOLUTE_TO_START: SensorEntityDescription(
+        key=TimerProperty.ABSOLUTE_TO_START,
+        translation_key=TimerProperty.ABSOLUTE_TO_START,
+    ),
+    TimerProperty.ABSOLUTE_TO_STOP: SensorEntityDescription(
+        key=TimerProperty.ABSOLUTE_TO_STOP,
+        translation_key=TimerProperty.ABSOLUTE_TO_STOP,
+    ),
+    TimerProperty.TARGET: SensorEntityDescription(
+        key=TimerProperty.TARGET,
+        translation_key=TimerProperty.TARGET,
+    ),
+}
```

Comment:
These do not have device classes or units of measurements, can you give an example of these states?

Reply:
I described their states in the text. 
They are just a time data, excluding date, and changed by user's setting.